src/graphics/text.rs

```diff
@@ -77,6 +78,7 @@ impl FontAtlas {
     ) -> Result<FontAtlas> {
         use rusttype as rt;
 
+        let em_pixels = font_size;
```

Comment:
Couldn't we just rename the "font_size" parameter to "em_pixels"?

Reply:
Yeah, we probably should.